cinema/views.py

```diff
@@ -1 +1,61 @@
-# write views here
+from rest_framework import viewsets
+
+from cinema.models import CinemaHall, Genre, Actor, Movie, MovieSession
+from cinema.serializers import (
+    CinemaHallSerializer, GenreSerializer,
```

Comment:
maybe it's better to import everything as a simple column (i saw that in a checklist)

Reply:
thanks)

Reply:
agree with @aLEKS-e3  :)